apps/backend/main.py

```
@@ -3,14 +3,28 @@
 from fastapi.middleware.cors import CORSMiddleware
 
+@asynccontextmanager
+async def lifespan(app: FastAPI):
+    # Startup: Nothing to do for now
+    yield
+    # Shutdown: Close AI client connections
+    try:
+        from utils.unified_ai_client import get_client
+        client = get_client()
+        await client.shutdown()
+    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)